sds_data_manager/lambda_code/SDSCode/download_query_api.py

```diff
@@ -33,19 +43,31 @@ def http_response(header_type="text/html", status_code=200, body="Success"):
 def lambda_handler(event, context):
     """This lambda handler checks if this file exists or not. If file doesn't exist, it
     gives back an error. Otherwise, it returns pre-signed s3 url that user can use to
-    donwload data from s3.
-
-    Args:
-        event (dict): input to lambda
-        context : This is not used.
+    download data from s3.
+
+    Parameters
+    ----------
+    event : dict
+        The JSON formatted document with the data required for the lambda function to process
+    context : LambdaContext
+        This object provides methods and properties that provide information about the invocation, function,
+        and runtime environment.
+
+    Returns
+    -------
+    dict
+        The response from the function which could either be a pre-signed S3 URL in case of successful
+        operation or an error message with corresponding status code in case of failure.
     """
-
     logger.info(event)
 
+    print(event)
```

Comment:
Are these for testing and accidentally left in?

Reply:
Yes. Geez. I thought that I got all of those. Fixed it.